controllers/ajax.py

```diff
@@ -50,13 +51,15 @@ def hsblog():
     div_id = request.vars.div_id
     event = request.vars.event
     course = request.vars.course
-    ts = datetime.datetime.now()
+    # Get the current time, rounded to the nearest second -- this is how time time will be stored in the database.
+    ts = datetime.datetime.utcnow()
```

Comment:
all controllers should use utcnow instead of now.  I seem to have misplaced a PR I had prepared that updates all the controllers so we are consistent.